### DISCUSSION TITLE
Add style while creating a new row

### DISCUSSION THREAD

**Discussion by CalibanAngel** [exceljs/exceljs#2499] [Ideas & Proposals]
Hi ! 
I'm adding row using array as the doc says using `worksheet.addRow([3, 'Sam', new Date()]);`, but now i would like to add style to my cell while adding the row. 
I would like to do something like that `worksheet.addRow([3, {text: 'Sam', style: {fill: {bgColor: rgb(42, 42, 42)}}, new Date()]);`. I don't know if it's possible and how to do it. 
Thanks !

**Comment by nik0145** [comment]
\+

**Comment by AlokSinghKumar** [comment]
+
